### PR TITLE
Including acronyms guidelines for camel casing in rule doc

### PR DESCRIPTION
**Pull request by veronicagg**
suggesting update to doc, including link to guidelines for acronyms.

### CODE REVIEW

File: docs/developer/validation-rules/camel-casing-validation-rules.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+This violation is flagged if a model definition's property name (DefinitionsPropertiesNamesCamelCase) or a request body parameter's property name (BodyPropertiesNamesCamelCase)  is not in `camelCase` format. This is because the model's properties are sent across the wire and must adhere to common Json conventions for naming. This implies that every property name must start with a lower cased letter and all subsequent words within the name must start with a capital letter. In cases where there are acronyms involved, a maximum of three contiguous capital letters are allowed (acronyms should not be longer that 2 characters - see https://msdn.microsoft.com/en-us/library/141e06ef(v=vs.71).aspx, third upper case could be start of next word). Eg: `redisCache`, `publicIPAddress`, `location` are valid, but `sampleSQLQuery` is not (must be renamed as `sampleSqlQuery`).
```
Comment (dsgouda) [inline docs/developer/validation-rules/camel-casing-validation-rules.md]: Just want to make sure we are not adhering to the MSFT Json naming conventions here. Can you confirm this.

Comment (veronicagg) [inline docs/developer/validation-rules/camel-casing-validation-rules.md]: @dsgouda which guidelines are you referring to?

Comment (dsgouda) [inline docs/developer/validation-rules/camel-casing-validation-rules.md]: [This](https://msdn.microsoft.com/en-us/library/ms229043(v=vs.110).aspx) one

Comment (veronicagg) [inline docs/developer/validation-rules/camel-casing-validation-rules.md]: In our case, we want to allow 2 letter acronyms to be in upper case.

**Pull request merged**